components/src/molecules/cookie-message/CookieMessage.js

```diff
@@ -39,16 +43,17 @@ export default {
     }
   },
   mounted () {
-    if (localStorage.getItem(this.dataType) !== 'closed') {
+    if (localStorage.getItem(this.storageName) !== 'closed') {
       this.isOpen = true
     }
   },
   methods: {
     closeBar () {
-      const focusable = document.querySelectorAll('button:not([disabled]), a[href], area[href] input:not([disabled]), select:not([disabled]), textarea:not([disabled]), *[tabindex]:not([tabindex="-1"]), object, embed, *[contenteditable]')
-
       this.isOpen = false
-      localStorage.setItem(this.dataType, 'closed')
+
+      const focusable = this.$root.$el.querySelectorAll('button:not([disabled]), a[href], area[href] input:not([disabled]), select:not([disabled]), textarea:not([disabled]), *[tabindex]:not([tabindex="-1"]), object, embed, *[contenteditable]')
+
+      localStorage.setItem(this.storageName, 'closed')
```

Comment:
We don't want to handle keeping status in local storage. We need just a component which we can use in application and `v-if` will decide if we show that or not.